python/cugraph/sampling/random_walks_wrapper.pyx

```diff
@@ -85,32 +87,71 @@ def random_walks(input_graph, start_vertices, max_depth):
                                                            graph_container,
                                                            <int*> c_start_vertex_ptr,
                                                            <int> num_paths,
-                                                           <int> max_depth))
+                                                           <int> max_depth,
+                                                           <bool> use_padding))
         else: # (edge_t == np.dtype("int64")):
             rw_ret_ptr = move(call_random_walks[int, long]( deref(handle_),
                                                            graph_container,
                                                            <int*> c_start_vertex_ptr,
                                                            <long> num_paths,
-                                                           <long> max_depth))
+                                                           <long> max_depth,
+                                                           <bool> use_padding))
     else: # (vertex_t == edge_t == np.dtype("int64")):
         rw_ret_ptr = move(call_random_walks[long, long]( deref(handle_),
                                                            graph_container,
                                                            <long*> c_start_vertex_ptr,
                                                            <long> num_paths,
-                                                           <long> max_depth))
+                                                           <long> max_depth,
+                                                           <bool> use_padding))
 
     
     rw_ret= move(rw_ret_ptr.get()[0])
     vertex_set = DeviceBuffer.c_from_unique_ptr(move(rw_ret.d_coalesced_v_))
     edge_set = DeviceBuffer.c_from_unique_ptr(move(rw_ret.d_coalesced_w_))
-    sizes = DeviceBuffer.c_from_unique_ptr(move(rw_ret.d_sizes_))
     vertex_set = Buffer(vertex_set)
     edge_set = Buffer(edge_set)
-    sizes = Buffer(sizes)
 
     set_vertex = cudf.Series(data=vertex_set, dtype=vertex_t)
     set_edge = cudf.Series(data=edge_set, dtype=weight_t)
-    set_sizes = cudf.Series(data=sizes, dtype=edge_t)
+
+    if not use_padding:
+        sizes = DeviceBuffer.c_from_unique_ptr(move(rw_ret.d_sizes_))
+        sizes = Buffer(sizes)
+        set_sizes = cudf.Series(data=sizes, dtype=edge_t)
+    else:
+        set_sizes = None
 
     return set_vertex, set_edge, set_sizes
-    
\ No newline at end of file
+
+
+def rw_path_retrieval(num_paths, sizes):
```

Comment:
I am confused about this function since there is no graph being populated. therefore I am expecting `call_rw_paths` it to be called inside `call_random_walks` in `cython.cu` or something like this but it doesn't seems to happen. How does this function help works ? (a description will help)

Reply:
it is the counterpart of rw_path in random_walks.py